integrations/gradient/src/gradient_haystack/embedders/gradient_document_embedder.py

```diff
@@ -31,15 +36,17 @@ def __init__(
         self,
         *,
         model_name: str = "bge-large",
-        batch_size: int = 100,
+        batch_size: int = 32768,
         access_token: Optional[str] = None,
         workspace_id: Optional[str] = None,
         host: Optional[str] = None,
+        progress_bar: bool = True,
```

Comment:
I think the convention is to name the variables with a "booleanish" prefix? `show_progress_bar`? `has_progress_bar`? `progress_bar` looks like it's expecting an actual object.